src/components/App/App.tsx

```diff
@@ -1,96 +1,63 @@
-import React from "react";
 import "./App.css";
 import NavBar from "../NavBar/NavBar";
-import RandomContainer from "../RandomContainer/RandomContainer";
-import SpecialtyContainer from "../SpecialtyContainer/SpecialtyContainer";
-import { fetchAllCocktails } from "../../apiCalls";
-import { Route } from "react-router-dom";
+import GinDeck from "../GinDeck";
 import MartiniDeck from "../MartiniDeck/MartiniDeck";
-import { Drink, Props, TypeState } from "../Types";
 import AmarettoDeck from "../AmarettoDeck";
 import DetailCards from "../DetailCards";
-import Error from "../Error/error";
 import AppContainer from "../AppContainer/AppContainer";
-import GinDeck from "../GinDeck";
+import Error from "../Error/error";
+import {FC, useState, useEffect} from "react"
+import { fetchAllCocktails } from "../../apiCalls";
+import { Route } from "react-router-dom";
+import { Drink } from "../Types";
 
-class App extends React.Component<{}, TypeState> {
-  constructor(props: Props) {
-    super(props);
-    this.state = {
-      drinkList: [],
-      id: "",
-      error: false,
-    };
-  }
+const App: FC = () => {
+  const [drinkList, setDrinkList] = useState <Drink[]> ([]);
```

Comment:
We have to remember to add state on hooks that invoke other functions in future projects as well.....easier said than done though